### PR TITLE
Fastimage

### PR DESCRIPTION
**Pull request by juananime**
Fixed not called onReourceLoad event on Android

### CODE REVIEW

File: android/src/main/java/com/dylanvann/fastimage/FastImageViewManager.java [hunk 1]
```diff
@@ -95,10 +95,15 @@ public boolean onResourceReady(
             if (!(target instanceof ImageViewTarget)) {
                 return false;
             }
+
+
```
Comment (Almouro) [inline android/src/main/java/com/dylanvann/fastimage/FastImageViewManager.java]: we don't need those spaces ;)

File: android/src/main/java/com/dylanvann/fastimage/FastImageViewManager.java [hunk 2]
```diff
@@ -95,10 +95,15 @@ public boolean onResourceReady(
...
             ImageViewWithUrl view = (ImageViewWithUrl) ((ImageViewTarget) target).getView();
             ThemedReactContext context = (ThemedReactContext) view.getContext();
             RCTEventEmitter eventEmitter = context.getJSModule(RCTEventEmitter.class);
             int viewId = view.getId();
+            if(resource.getMinimumHeight() * resource.getMinimumWidth() < 10){
+                eventEmitter.receiveEvent(viewId, REACT_ON_ERROR_EVENT, null);
```
Comment (Almouro) [inline android/src/main/java/com/dylanvann/fastimage/FastImageViewManager.java]: not sure we want to send onLoad event then

**Review by Almouro — CHANGES_REQUESTED** [review]

File: android/src/main/java/com/dylanvann/fastimage/FastImageViewManager.java (hunk 2 above)
Comment (juananime) [reply android/src/main/java/com/dylanvann/fastimage/FastImageViewManager.java]: Then we will have same problem right? we need the load event to load always right?

File: android/src/main/java/com/dylanvann/fastimage/FastImageViewManager.java (hunk 1 above)
Comment (juananime) [reply android/src/main/java/com/dylanvann/fastimage/FastImageViewManager.java]: removing straight away :)

File: android/src/main/java/com/dylanvann/fastimage/FastImageViewManager.java (hunk 2 above)
Comment (juananime) [reply android/src/main/java/com/dylanvann/fastimage/FastImageViewManager.java]: gonna quickly test, give me 5 min

Comment (Almouro) [reply android/src/main/java/com/dylanvann/fastimage/FastImageViewManager.java]: https://facebook.github.io/react-native/docs/image.html#onload

If error -> onLoadEnd should be called but not onLoad

**Review by Almouro — APPROVED** [review]

**Pull request merged**